### PR TITLE
media-gfx/imv: 4.2.0 bump

### PR DESCRIPTION
**Pull request by ionenwks**
Gave this a full checkup + bump, most of the meson migration was already done in -9999

Was tested on wayland, x11, amd64, and x86.

Given took a liking to this package, also adding myself as maintainer (first time) and will be properly looking after it.

### CODE REVIEW

**Comment by ionenwks** [comment]
Sneaked in an extra patch to fix a flickering issue on wayland under some configurations.

File: media-gfx/imv/imv-4.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	fi
+
+	# glu isn't used by anything
+	sed -i "/dependency('glu')/d" meson.build || die
+}
+
+src_configure() {
+	local WINDOWS=all
+	use X || WINDOWS=wayland
+	use wayland || WINDOWS=x11
```
Comment (juippis) [inline media-gfx/imv/imv-4.2.0.ebuild:72]: We use lowercase lettering for local variables, and uppercase for global one to separate them.

File: media-gfx/imv/imv-4.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	local emesonargs=(
+		$(meson_feature freeimage)
+		$(meson_feature gif libnsgif)
+		$(meson_feature heif libheif)
+		$(meson_feature jpeg libjpeg)
+		$(meson_feature png libpng)
+		$(meson_feature svg librsvg)
+		$(meson_feature test)
+		$(meson_feature tiff libtiff)
+		-Dwindows=$WINDOWS
```
Comment (juippis) [inline media-gfx/imv/imv-4.2.0.ebuild:83]: And curly brackets for variables, 
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101

**Review by juippis — APPROVED** [review]

File: media-gfx/imv/imv-4.2.0.ebuild (hunk 2 above)
Comment (ionenwks) [reply media-gfx/imv/imv-4.2.0.ebuild:83]: Thanks, was like that in the original -9999 ebuild and overlooked it.

Done.

File: media-gfx/imv/imv-4.2.0.ebuild (hunk 1 above)
Comment (ionenwks) [reply media-gfx/imv/imv-4.2.0.ebuild:72]: Done.

**Comment by ionenwks** [comment]
Oh wait, had already changed, it. Thanks :)

**Pull request closed without merging**